wagtail/admin/forms/collections.py

```diff
@@ -68,6 +68,7 @@ def label_from_instance(self, obj):
 
 class CollectionForm(forms.ModelForm):
     parent = CollectionChoiceField(
+        label=_("Parent"),
```

Comment:
Looks like this file imports ``gettext`` as ``_``. In this instance, we need to use ``gettext_lazy`` or this would be evaluated on startup.

Reply:
Ah nice catch, completely missed that.

Thank you for the review!